package.json

```diff
@@ -22,12 +24,15 @@
     "url": "git@github.com:bjrmatos/jsreport-jade.git"
   },
   "dependencies": {
+    "in-publish": "2.0.0",
     "object-assign": "4.0.1",
     "toner-jade": "1.0.0"
   },
   "devDependencies": {
+    "brace": "0.8.0",
```

Comment:
@pofider same question, any reason to have `brace` in `devDependencies` and not in `dependencies`? seems like it should be in `dependencies` but you know how `jsreport-studio-build` works 😄